src/Pact/Types/Gas.hs

```diff
@@ -164,9 +164,9 @@ instance FromJSON GasLimit where
 instance Wrapped GasLimit
 
 data GasModel = GasModel
-  { gasModelName :: Text
-  , gasModelDesc :: Text
-  , runGasModel :: Text -> GasArgs -> Gas
+  { gasModelName :: !Text
+  , gasModelDesc :: !Text
+  , runGasModel :: !(Text -> GasArgs -> Gas)
```

Comment:
unpacking our `Text` and `Code` fields explicitly might be good too.

Reply:
Not sure how much it would help. Let's keep those finer and more low-level optimizations for another PR.